src/pages/ReportChangeWorkspacePage.tsx

```diff
@@ -45,7 +45,9 @@ function ReportChangeWorkspacePage({report}: ReportChangeWorkspacePageProps) {
                 return;
             }
             Navigation.goBack(ROUTES.REPORT_WITH_ID.getRoute(reportID));
-            if (isIOUReport(reportID) && isPolicyMember(currentUserLogin, policyID)) {
+            if (isIOUReport(reportID) && isPolicyAdmin(getPolicy(policyID))) {
```

Comment:
This should only be called if the submitter is not a member of the workspace
```suggestion
            if (isIOUReport(reportID) && isPolicyAdmin(getPolicy(policyID)) && report.ownerAccountID && !isPolicyMember(getLoginByAccountID(report.ownerAccountID), policyID)) {
```